app/controllers/api/v1/api_controller.rb

```diff
@@ -6,6 +6,12 @@ class ApiController < ApplicationController
 
     private
 
+    def respond_with(serializer)
+      respond_to do |format|
+        format.json { render json: serializer }
+      end
+    end
+
```

Comment:
I am wary of adding an extra dependency (responders gem) just to get 1 method which is all we need right now.
But you do have a point about using a "standard" rails method name so how about:

```
def respond_as_json(serializer)
  ...
end
```

?